app/grants/views.py

```diff
@@ -1351,8 +1361,10 @@ def grant_new(request):
         'conf_time_spread': conf_time_spread(),
         'gas_advisories': gas_advisories(),
         'trusted_relayer': settings.GRANTS_OWNER_ACCOUNT,
-        'grant_types': GrantType.objects.all()
+        'grant_types': GrantType.objects.all(),
+        'data': project_data
```

Comment:
please use a better name than data, could be the same 'project_data'

Reply:
Done